corehq/apps/app_manager/views/modules.py

```diff
@@ -488,6 +489,11 @@ def rename_action_case_type(mod):
             except ModuleNotFoundException:
                 messages.error(_("Unknown Module"))
 
+    if should_edit('incl_form_ids') and isinstance(module, ShadowModule):
+        incl = request.POST.getlist('incl_form_ids') or []
+        excl = {f.unique_id for f in module.source_module.get_forms()} - set(incl)
+        module.excluded_form_ids = list(excl)
```

Comment:
Looks like in the edge case where someone adds forms to the source module in one window and saves the config of the shadow module in another the new forms (which weren't showed to the user in the shadow modules UI) would default to being excluded rather than included. Not sure whether or not that's the optimal behavior, but it's almost certainly too rare to worth changing; just wanted to flag that that's how it'd work, in case that causes you to think of something.